scripts/publish.sh

```diff
@@ -19,10 +19,13 @@ echo "============================"
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-NPM_VERSION=$2
+NPM_VERSION=$(sh scripts/prisma-version.sh "$CHANNEL")
```

Comment:
On trying this file out, I get the following error, rightly so, because the variable is called `RELEASE_CHANNEL` in this file. 

![image](https://user-images.githubusercontent.com/746482/84916304-8a218380-b0db-11ea-94b4-d580778682be.png)

Whether you want to apply this suggestion or change the variable name is up to you. 

```suggestion
NPM_VERSION=$(sh scripts/prisma-version.sh "$RELEASE_CHANNEL")
```